modules/persistent-storage-local-discovery.adoc

```diff
@@ -36,20 +36,19 @@ Use the `LocalVolumeSet` object with caution. When you automatically provision p
 
 . To enable automatic discovery of local devices from the web console:
 
-.. In the _Administrator_ perspective, navigate to *Operators* -> *Installed Operators* and click on the *Local Volume Discovery* tab.
+.. Click *Operators* -> *Installed Operators*.
 
-.. Click *Create Local Volume Discovery*.
+.. In the `openshift-local-storage` namespace, click *Local Storage*.
 
-.. Select either *All nodes* or *Select nodes*, depending on whether you want to discover available disks on all or specific nodes.
-+
-[NOTE]
-====
-Only worker nodes are available, regardless of whether you filter using *All nodes* or *Select nodes*.
-====
-+
-.. Click *Create*.
+.. Click the *Local Volume Discovery* tab.
+
+.. Click *Create Local Volume Discovery* and select either *Form view* or *YAML view*.
+
+.. Configure the `LocalVolumeDiscovery` object parameters.
```

Comment:
Is there any place where the parameters are documented? If there is one, it should be linked from Additional resources.

Reply:
The `LocalVolumeDiscovery` object parameters are not documented yet. But I agree with you, it will be useful to have this section linked in the Additional resources.